### PR TITLE
Update getting-started-chunks.md

### PR DESCRIPTION
**Pull request by brianpham**
**What this PR does**:
Add missing instructions to match setting up prometheus

**Which issue(s) this PR fixes**:
No issue created

### CODE REVIEW

**Comment by brianpham** [comment]
Not sure if I need to update CHANGELOG for this change?

File: docs/getting-started/getting-started-chunks.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Cortex can also make use of external memcacheds for caching and although these a
 For simplicity and to get started, we'll run it as a [single process](../configuration/single-process-config.md) with no dependencies:
 
 ```sh
+$ git clone git@github.com:cortexproject/cortex.git
```
Comment (pstibrany) [inline docs/getting-started/getting-started-chunks.md:24]: I would suggest to use HTTP-based clone instead, as it doesn't depend on `ssh` and can use proxies.

```suggestion
$ git clone https://github.com/cortexproject/cortex.git
```

Comment (brianpham) [reply docs/getting-started/getting-started-chunks.md]: That make sense. Fixing.

**Comment by pstibrany** [comment]
Oh, I just realized it says “Prometheus“ instead of Cortex! My bad. Do you want to send PR to fix it? (If not, I will on Monday)

**Comment by brianpham** [comment]
Ahh silly mistake. Just created a new PR.

**Pull request merged**